pkg/view/router.go

```diff
@@ -471,11 +491,12 @@ func New(
 				return
 			}
 
-			m.HideSidebar = true
-			err = gv.Render(w, http.StatusOK, "pages/request/success", m)
+			err = storageServices.Database.DeleteConnectionRequest(r.Context(), connReq.ID)
 			if err != nil {
-				http.Error(w, err.Error(), http.StatusInternalServerError)
+				log.Err(err).Msg("failed to delete connection request")
 			}
+
+			http.Redirect(w, r, "/dashboard/request/success", http.StatusFound)
```

Comment:
There is an edge case where the user hits the back button, then they end up on this page. However it correctly throws an error saying "connection request not found" so i think we can ignore that for now.